src/controllers/ChatController.ts

```diff
@@ -0,0 +1,42 @@
+import API, {CharsApi, chatAdd} from '../api/CharsApi';
+import store from '../utils/Store';
+
+export class ChatController {
+  private readonly api: CharsApi;
+
+  constructor() {
+    this.api = API;
+  }
+
+  async fetchChats() {
+    const chats = await this.api.read();
+
+    try {
+      store.set('chats', chats);
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  async createChat(data: {title: string}) {
+    await this.api.create(data);
+    this.fetchChats();
```

Comment:
fetchChats - асинхронный метод и его так же нужно вызывать через await
```suggestion
    await this.fetchChats();
```